tests/data/runFull/in/tables/nullable.csv

```diff
@@ -8,3 +8,4 @@
 "6","Petr Simecek","yes",""
 "7","Jakub Matejka","yes",""
 "8","Ondrej Popelka","no",""
+"9","","no",""
```

Comment:
Najprv som pridal tento test, ... tj. zapisuje sa prazdny string do `NOT NULL` stlpca ... tak sa mi podarilo nasimulovat rovnaku chybu ako v ZD tickete.